### PR TITLE
Update "examples" content to match other resource yaml files for better parsing.

### PR DESCRIPTION
**Pull request by jtonello**
### Description

Update "examples" content to match other resource yaml files for better parsing.

### Issues Resolved
File not parsing correctly in PHP handler

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: data/infra/resources/chocolatey_config.yaml [hunk 1]
```diff
@@ -41,7 +41,18 @@ properties_list:
...
+    value 'C:\temp\boco'
+  end
+  ```
+  **Unset a Chocolatey config**:
+  
+  ```
+  chocolatey_config 'BogusConfig' do
+    action :unset
+  end
+  ```
```
Comment (IanMadd) [inline data/infra/resources/chocolatey_config.yaml]:
````suggestion
  **Set the Chocolatey cacheLocation config**:

  ```ruby
  chocolatey_config 'Set cacheLocation config' do
    config_key 'cacheLocation'
    value 'C:\temp\choco'
  end
  ```

  **Unset a Chocolatey config**:

  ```ruby
  chocolatey_config 'BogusConfig' do
    action :unset
  end
  ```
````

Comment (IanMadd) [reply data/infra/resources/chocolatey_config.yaml]: The examples aren't quite correct ☝️

**Comment by tas50** [comment]
@jtonello This content is actually generated automatically from:

https://github.com/chef/chef/blob/master/lib/chef/resource/chocolatey_config.rb

**Comment by jtonello** [comment]
Ah, thanks @tas50.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**